src/realm-roles/AssociatedRolesModal.tsx

```diff
@@ -43,15 +43,38 @@ export const AssociatedRolesModal = (props: AssociatedRolesModalProps) => {
     return _.sortBy(rolesList, (role) => role.name?.toUpperCase());
   };
 
-  const loader = async () => {
-    const allRoles = await adminClient.roles.find();
+  const loader = async (first?: number, max?: number, search?: string) => {
```

Comment:
```suggestion
  const loader = async (_first?: number, _max?: number, search?: string) => {
```
first and max are not used